packages/eslint-plugin-wpcalypso/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+#### next (…)
```

Comment:
> (…)

🍒

Reply:
I intentionally left this to be filled in for next release. I'm not sure what 🍒 means though 😆

Reply:
Oh sorry, maybe it's a cultural proverb, "as a cherry on top" 🤔  😅 

I just liked you used … in a PR about …

Reply:
😎 

---

🍒 
👇 
🍨 ✨